applications/callflow/src/module/cf_ring_group.erl

```diff
@@ -134,6 +134,11 @@ start_builder(EndpointId, Member, Call) ->
       end
      ).
 
+-spec member_active(kz_json:object()) -> boolean().
+member_active(Member) ->
+    DisableUntil = kz_json:get_value(<<"disable_until">>, Member, 0),
+    calendar:datetime_to_gregorian_seconds(calendar:universal_time()) > DisableUntil.
```

Comment:
Ok, I see now. Is the plan to allow users to customize how long to disable? Otherwise it seems like this should be a binary decision and not a temporal one. Thoughts?